doc/release-tasklist

```diff
@@ -71,6 +71,51 @@
        dealii@googlegroups.com
 
 
+Announcement for 1.5.0 (February 18, 2017)
+-----------------------------------------
+We are pleased to announce the release of ASPECT 1.5.0. ASPECT is the Advanced
+Solver for Problems in Earth's ConvecTion. It uses modern numerical methods such
+as adaptive mesh refinement, multigrid, and a modular software design to
+provide a fast, flexible, and extensible mantle convection solver. ASPECT is
+available from
+
+                   https://aspect.dealii.org/
+
+and the release is available from
+
+        https://github.com/geodynamics/aspect/releases/tag/v1.5.0
+
+This release includes the following changes:
+
+- New: Choice between different formulations for the governing equations
+  including Boussinesq and anelastic liquid approximation
+- New: Melt transport (two-phase flow)
+- New: Nondimensional material model for incompressible (using the Boussinesq
+  approximation) and compressible computations (with ALA or TALA) for
+  nondimensionalized problems. This can be used for benchmark problems like
+  Blankenbach, King, etc..
```

Comment:
be consistent with periods at the end of each bullet points (either with or without; the last one always with).